js/client/modules/@arangodb/testutils/process-utils.js

```diff
@@ -1350,12 +1397,16 @@ function executeArangod (cmd, args, options) {
 function getSockStat(arangod, options, preamble) {
   if (options.getSockStat && (platform === 'linux')) {
     let sockStat = preamble + arangod.pid + "\n";
-    try {
-      sockStat += fs.read("/proc/" + arangod.pid + "/net/sockstat");
-      return sockStat;
-    }
-    catch (e) {/* oops, process already gone? don't care. */ }
+    sockStat += getSockStatFile(arangod.pid);
+    return sockStat;
+  }
+}
+function getSockStatFile(pid) {
+  try {
+    let sockStat = fs.read("/proc/" + pid + "/net/sockstat");
+    return sockStat;
   }
+  catch (e) {/* oops, process already gone? don't care. */ }
```

Comment:
could be slightly simplified, but it is not important
```suggestion
  try {
    return fs.read("/proc/" + pid + "/net/sockstat");
  } catch (e) {/* oops, process already gone? don't care. */ }
```